cmd/eulabeia-director/main.go

```diff
@@ -43,8 +44,12 @@ func main() {
 	config.OverrideViaENV(configuration)
 	server := configuration.Connection.Server
 
-	log.Println("Starting director")
-	client, err := mqtt.New(server, *clientid, "", "", nil, []connection.Preprocessor{scan.StartMegaScan{}})
+	prepare_topic := func(aggregate_name string) string {
```

Comment:
This could be a general helper function for all `main.go`'s, or even more generic, like
```
func prepareTopic(context string, aggregate string, event string, destination string) {
  // yada yada
}

// more wrapper for e.g. director, cmd 
func prepareDirectorCmdTopic ...
```

Maybe?

Reply:
maybe :) can look into it